### PR TITLE
Refactor: The big namespace rename of 2022

### PR DESCRIPTION
**Pull request by uxmal**
The project has grown big, and some folders and namespaces need to be corrected to make it easier to navigate the project.

### CODE REVIEW

**Comment by uxmal** [comment]
Let me know if you have concerns about the big renaming.

File: src/UnitTests/Core/Serialization/Json/JsonProjectSerializerTests.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace Reko.UnitTests.Analysis
+namespace Reko.UnitTests.Decompiler.Analysis
```
Comment (ptomin) [inline src/UnitTests/Core/Serialization/Json/JsonProjectSerializerTests.cs:35]: Maybe should be `Reko.UnitTests.Core.Serialization.Json`

**Review by ptomin — APPROVED** [review]
Just one little note.

File: src/UnitTests/Core/Serialization/Json/JsonProjectSerializerTests.cs (hunk 1 above)
Comment (uxmal) [reply src/UnitTests/Core/Serialization/Json/JsonProjectSerializerTests.cs:35]: Good catch!

**Pull request merged**